Environments permissions about page (#5618)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/secure/environment-permissions.md

---
title: "About environment-level permissions"
id: environment-permissions
description: "About environment-level permissions to protect your information"
sidebar_label: "Environment-level permissions"
pagination_next: null
pagination_prev: null
---

# About environment-level permissions <Lifecycle status='beta' />

:::note

This is a beta feature available to select dbt Cloud Enterprise customers. If you are interested in beta testing this feature, please contact your account manager.

:::

> Environment-level permissions give dbt Cloud admins the ability to grant write permission to groups and service tokens for specific [environment types](/docs/dbt-cloud-environments) within a project. Granting access to an environment give users access to all environment-level write actions and resources associated with their assigned roles. For example, users with a Developer role can create and run jobs within the environment(s) they have access to. For all other environments, that same user will have read-only access. 

Reviewer: update "user" to "users"

> Environment-level permissions give dbt Cloud admins the ability to grant write permission to groups and service tokens for specific [environment types](/docs/dbt-cloud-environments) within a project. Granting access to an environment give users access to all environment-level write actions and resources associated with their assigned roles. For example, users with a Developer role can create and run jobs within the environment(s) they have access to. For all other environments, that same user will have read-only access. 

Author: ```suggestion
Environment-level permissions give dbt Cloud admins the ability to grant write permission to groups and service tokens for specific [environment types](/docs/dbt-cloud-environments) within a project. Granting access to an environment give users access to all environment-level write actions and resources associated with their assigned roles. For example, users with a Developer role can create and run jobs within the environment(s) they have access to. For all other environments, those same users will have read-only access. 
```
